integreat_compass/cms/models/users/role.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.auth.models import Group
+from django.db import models
+from django.utils import translation
+from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+
+from ...constants import roles
+from ..abstract_base_model import AbstractBaseModel
+
+
+class Role(AbstractBaseModel):
```

Comment:
I'm not even sure if this project needs the proxy model `Role`, maybe the default Django `Group` is enough?

Reply:
We should probably discuss roles tomorrow in the team call. I'm more than open to a simpler system, as you mentioned above. We had not really talked about it previously, I just felt the need to include at least a rough draft with this PR.

Btw, thanks for the (preliminary) review, I know it's a frustratingly large PR, sorry about that.